### DISCUSSION TITLE
[Application Development] l1gasFee ecotone calculation

### DISCUSSION THREAD

**Discussion by wew6d48g** [ethereum-optimism/developers#288] [Application Development]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

i looked at the updated ecotone l1 fee calculation but i am unsure where to get certain values
https://docs.optimism.io/builders/dapp-developers/transactions/fees#ecotone

where do i get the current base fee scalar and blob_base_fee_scalar?
weighted_gas_price = 16 * base_fee_scalar * base_fee + blob_base_fee_scalar * blob_base_fee

**Comment by hash1go** [comment]
I think these two document pages will be helpful.

https://docs.optimism.io/builders/notices/ecotone-changes#for-chain-operators
https://docs.optimism.io/builders/chain-operators/management/blobs
